doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 Intel Corporation.
+# SPDX-License-Identifier: BSD-3-Clause
+# This script is aim to setup or upgrade sos & uos automatically.
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+function print_help()
+{
+echo "Usage:"
+echo "Launch this script as: ./flash.sh -t 28100"
+echo "-t to specify clear linux version for upgrading"
+echo "-p to specify a proxy server"
+echo "-m to use swupd mirror url"
+echo "-u to upgrade uos, if you just upgrade sos using this script, you'd better to manually reboot sos then continue to use this function."
+echo "-s, skip downloading uos; if enabled, you have to download uos img firstly before upgrading, default is off"
+exit 1
+}
+
+function upgrade_sos()
+{
+# setup mirror, proxy env if specified parameters to p & m opt.
+swupd mirror -u
+if [[ -n $mirror ]]; then
+    echo "Add swupd mirror..."
+    swupd mirror -s $mirror
+fi
+unset https_proxy
```

Comment:
We should only do this if a proxy is provided. Usually, one expects proxy to be transparently. What I mean by that is that if the system knows the proxy information (which should represent 90+% of the cases), then we shouldn't use this setting. This should only be needed where the proxy setting were not automatically determined.

Reply:
Also, why do you unset this twice (it's also done below in line 108)

Reply:
Thanks for the comments, it should be removed.